setup.py

```diff
@@ -6,7 +6,7 @@
 import versioneer
 
 
-include_dirs = ['ffld2', 'ffld2/lib']
+include_dirs = ['cyffld2/ffld2', 'cyffld2/ffld2/lib']
```

Comment:
This may be required but I'll have to check why it isn't failing for CI right now.